lib/assessment.rb

```diff
@@ -13,6 +14,8 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
+count = count.{ |memo, enum|}
+    memo + 1
```

Comment:
Not the right direction on this one. Here's a possible solution.
`enum.reduce {|memo, elem| memo + elem }`
Or even more tersely, 
`enum.reduce(&:+)`